recipe/meta.yaml

```diff
@@ -1,28 +1,30 @@
 {% set name = "openff-nagl-models" %}
-{% set version = "0.3.0" %}
+{% set version = "0.4.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
-  sha256: 62a157513a668bf8200932f59b1a5608c08371b63a4c8293ba46b2587385144b
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: 3bda84bce1b6cf94285d9ba4ad2eb51629fb89686304c1b22c476ccb523ed80f
 
 build:
   script: {{ PYTHON }} -m pip install . -vv
-  number: 1
+  number: 0
   noarch: python
 
 requirements:
   host:
-    - python >=3.9
+    - python 3.10
     - pip
-    - versioneer-518
+    - versioningit
     - setuptools
   run:
-    - python >=3.9
+    - python >=3.10
+    - platformdirs
     - importlib_resources
```

Comment:
I thought we dropped this in the source code? We should, if at all possible

Reply:
While we no longer fetch from GH release assets, caching is used for models fetched from zenodo. So we use platformdirs to inform the location of that cache. https://github.com/search?q=repo%3Aopenforcefield%2Fopenff-nagl-models%20platformdirs&type=code